### PR TITLE
Add more external links

### PR DESCRIPTION
**Pull request by MicheleK17**
Ticket: #248
Change-Id: I870acc82250641cdfcf2b740febb727b055226de

### CODE REVIEW

File: src/assets/external_links.json [hunk 1]
```diff
@@ -23,9 +35,17 @@
                 {
                     "Text": "ZoonosenMonitoring",
                     "Link": "https://www.bvl.bund.de/DE/Arbeitsbereiche/01_Lebensmittel/01_Aufgaben/02_AmtlicheLebensmittelueberwachung/06_ZoonosenMonitoring/lm_zoonosen_monitoring_node.html"
+                },
+                {
+                    "Text": "AR",
+                    "Link": "https://www.bmel.de/DE/themen/tiere/tierarzneimittel/lagebild-antibiotikaeinsatz-bei-tieren.html"
+                },
+                {
+                    "Text": "Antibiotikaminimierungskonzepts",
```
Comment (GerberI) [inline src/assets/external_links.json:44]: Is the trailing 's' intentional?

File: src/locales/de/ExternLinks.json [hunk 2]
```diff
@@ -6,12 +6,17 @@
...
+        "EU13": "Durchführungsbeschluss (EU) 2013/652/EU",
+        "EU20": "Durchführungsbeschluss (EU) 2020/1729"
     },
     "Reports": {
         "Title": "Berichte",
         "ZoonosenBfR": "Zoonosenberichterstattung durch das BfR",
-        "ZoonosenMonitoring": "Berichte zum Zoonosen-Monitoring"
+        "ZoonosenMonitoring": "Berichte zum Zoonosen-Monitoring",
+        "AR": "Lagebild zu Antibiotikaresistenzen und -einsatz im Bereich Tierhaltung und Lebensmittelkette",
+        "Antibiotikaminimierungskonzepts": "Bericht des BMEL über die Evaluierung des Antibiotikaminimierungskonzepts der 16. AMG-Novelle"
```
Comment (GerberI) [inline src/locales/de/ExternLinks.json:19]: I would spare a '-' here: Antibiotika-Minimierungskonzepts

**Review by GerberI — COMMENTED** [review]
Commit: Add more external links
Missing English translation

File: src/locales/de/ExternLinks.json (hunk 2 above)
Comment (MicheleK17) [reply src/locales/de/ExternLinks.json:19]: The text is given by Katja.

File: src/assets/external_links.json (hunk 1 above)
Comment (MicheleK17) [reply src/assets/external_links.json]: I changed the key to an abbreviation.

**Review by GerberI — APPROVED** [review]
Commit: Add english translation

**Comment by GerberI** [comment]
Ready to merge

**Pull request merged**